src/roman_datamodels/testing/factories.py

```diff
@@ -663,18 +664,50 @@ def create_wfi_img_photom_ref(**kwargs):
     roman_datamodels.stnode.WfiImgPhotomRef
     """
     raw_dict = {
+        "F062":
+            {"photmjsr": (10 * np.random.random() * u.MJ / u.sr),
```

Comment:
Can this be something like,
"photmjsr": (1.0e-15 * np.random.random() * u.MJ / u.sr
and
"uncertainty": 1.0e-15 * np.random.random() * u.MJ / u.sr
To get the magnitude more or less correct.

Reply:
Updated - also updated the maker utility to match.